dcpy/connectors/edm/publishing.py

```diff
@@ -380,21 +381,31 @@ def _assert_gis_dataset_exists(name: str, version: str):
 
 
 def get_latest_gis_dataset_version(dataset_name: str) -> str:
-    """Get latest version of GIS-published dataset in edm-publishing/datasets
-    from metadata of dataset in 'staging' folder"""
-    ## TODO
-    ## might make more sense to just list folders, grab "max" version
-    ## by that logic, should likely try to parse str version to dcpy version object
-    ## so we don't have strange cases of an unexpected subfolder (that starts with an _)
-    ## this would let us not have to supply versions for geosupport datasets ("24a", etc)
-    metadata = s3.get_metadata(BUCKET, _gis_dataset_path(dataset_name, "staging"))
-    version = metadata.last_modified.strftime("%Y%m%d")
+    """
+    Get latest version of GIS-published dataset in edm-publishing/datasets
+    assuming versions are sortable
+    """
+    gis_version_formats = [r"^\d{2}[A-Z]$", r"^\d{8}$"]
```

Comment:
Should we move this over to dcpy.utils.versions?

Reply:
Maybe. We need to make that more consistent and really implement it across things, so I'd rather just leave for now